vars/durationMinutes.groovy

```diff
@@ -0,0 +1,24 @@
+// vars/durationMinutes.groovy
+
+  /**
+   * durationMinutes step method
+   *
+   * This does not create the file, it is so the methods
+   * that create / fetch / read the STAGE status file all use
+   * the same name and it is properly URLEncoded.
+   *
+   * It does make sure that the directory for the stage
+   * status file exists.
+   *
+   * @param startDate, Date object with start time.
+   * @param endDate, Date object with end time, default current date time.
+   * returns: Duration time in minutes.
+   */
+
+int call(Date startDate, Date endDate=null) {
+    if (endDate == null) {
+        endDate = new Date()
+    }
+    int delta = (endDate.getTime() - startDate.getTime())/1000/60
```

Comment:
It's a pity that we are rounding this up to minutes.  I thought our choice to use minutes was because it was somehow easier.

IMO, input data should always be as precise as it can be, and it's up to the consumer to decide what level of precision they want to distill the data down to.

We might one day come to regret that we don't know how many seconds a stage is taking.